data/remnant missions.txt

```diff
@@ -1713,3 +1717,151 @@ mission "Remnant: Broken Jump Drive job"
+			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`
+			`	You tell Plume about how you visited Nenia and had a threatening encounter with a void sprite, and offer him the recording and container with the tentacle. Plume recoils from you in disgust. "After all our work rebuilding trust with the Archon and the void sprites, you go and risk it for... what, exactly? Sightseeing? Thrill of the hunt?" His chant settles into an angry tone. "Leave the recording on the console. I want no part of that tentacle, though."`
+			`	As you are leaving Plume's lab a technician pulls you aside. "Our records mention that the tentacles had some interesting properties that were useful in combat. If you are interested, meet me in the spaceport in a few hours."`
+
+
+
+mission "Remnant: Face to Maw 2"
+	name "Take the slime sample to Viminal"
+	description "The void sprite slime has properties of interest to a team of engineers on <planet>."
+	to offer
+		has "Remnant: Face to Maw 1: done"
+	source "Aventine"
+	destination "Viminal"
+	cargo "slime sample" 1
+	on offer
+		conversation
+			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a datapad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that," he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though.`
+			`	"Be that as it may be, the more immediate reward for your curiosity is in this sample," he trills with excitement. "Several of our labs on Viminal have been working on the development of a new generation of ship technology, and this substance contains some chemicals that could be a key component. I have already transmitted the data to them, but they would like to compare with the original. Could you take this sample to them?"`
+			choice
+				`	"Yes, I can."`
+					accept
+				`	"I would rather not."`
+			`	Plume stoically accepts that with a nod. "That is your choice. I will arrange for another ship to transport it."`
+			decline
+	on accept
+		event "remnant: penguin"
+	on complete
+		log "People" "Taely" `One of Taely's main priorities is improving the maneuverability characteristics of Remnant Ships. To this end she has been working on building a new prototype ship based on the void sprites.`
+		"remnant taely" ++
```

Comment:
Watch out with this. This is the scenario that tehhowch talks about [here](https://github.com/endless-sky/endless-sky/issues/3746#issuecomment-525282390) where a condition is altered and then immediately used in the next conversation.

Reply:
How should I change it (or does it need to be changed?) The intent is to have one line of dialogue displayed if this is the first time the player has met Taely; and another if it is the second or greater. Which seems to work as intended, unless someone quits right at that moment.

The only way around this that I can think of is moving the log and Taely variable to an invisible mission that triggers and resolves silently after this mission has been marked as "done." And then doing the same for every other mission that has this same branch in it.

I don't really want to expand this PR to do a revamp of other conversations; so if this is the best solution, I would suggest that this PR gets merged with the conversation as-is, and a follow-up PR changes all the conversation branches together.

On the other hand, I'm not too concerned if someone gets two slightly different conversations if they quit right there. I can just imagine the players "What the... Does Taely remember me from the alternate timeline where she already had this conversation with me?"